linkatos/message.py

```diff
@@ -21,3 +25,58 @@ def has_a_yes(message):
     Returns True if it matches the yes regex
     """
     return yes_re.search(message) is not None
+
+
+def has_a_no(message):
+    """
+    Returns True if it matches the no regex
+    """
+    return no_re.search(message) is not None
+
+
+def parse(slack_rtm_output):
+    """
+        The Slack Real Time Messaging API is an events firehose.
+        this parsing function returns None unless:
+        1. someone posts a url starting with httpS?//
+           in this case it returns the url and a out_type = 'url'
+        2. a yes or a no after a url
+           in this case it returns True for a yes, False for a no
+        The function also extracts the name of the channel
```

Comment:
I would like to know what is encoded in the returning tuple `(None, None, None)` is extremely cryptic.  So, either add some comments explaining the purpose of each element or change to a dictionary where the keys explain the purpose.